internal/integration/api/selinux.go

```diff
@@ -152,6 +153,13 @@ func (suite *SELinuxSuite) checkFileLabels(nodes []string, expectedLabels map[st
 			suite.T().Skip("skipping SELinux test since SELinux is disabled")
 		}
 
+		// TODO: do not skip the test if system extensions are enabled
```

Comment:
probably easier to check if either `extensions.qemu` or `extensions.nvidia` is set

Reply:
But what if we just run locally? Well, for CI that'd suffice

Reply:
same, those tests gets enabled only if the flag is passed